waku/v2/node/waku_node.nim

```diff
@@ -308,10 +304,10 @@ proc unsubscribe*(node: WakuNode, topic: PubsubTopic, handler: WakuRelayHandler)
     error "Invalid API call to `unsubscribe`. WakuRelay not mounted."
     return
 
-  debug "unsubscribe", oubsubTopic= topic
+  debug "unsubscribe", pubsubTopic=topic
 
   let wakuRelay = node.wakuRelay
-  wakuRelay.unsubscribe(@[(topic, handler)])
+  wakuRelay.unsubscribe(topic)
```

Comment:
good catch. this was a leftover. i have replaced unsubscribeAll by unsubcribe. So whenever we call now unsubscribe, we unsubscribe all handlers from the topic, meaning 100% unsubscribing to the topic.

unsubscribing individual handlers seemed very specific and since now we wrap the handler to only accept a WakuMessage, it wasnt trivial to adapt it.

If in the future we need to unsubscribe individual handlers, we can revisit this, but imho it would be better to incentivice people to use a single handler that does everything.

fixed dc70df97f859a4fa907db0be13bfd56d7a23dce8